app/assets/javascripts/heavens_door.js

```diff
@@ -47,12 +47,12 @@
   }
 
   Array.from(document.getElementsByTagName('form')).forEach(form => {
-    form.addEventListener('submit', e => {
+    form.addEventListener('submit', () => {
       if (sessionStorage.heavensDoor) {
         Array.from(form.querySelectorAll('input,textarea')).forEach(el => {
           let target = labelIdForElement(el) || el.id;
 
-          if ((el.type == 'text') || (el.type == 'textarea') || (el.type == 'search') || (el.type == 'number') || (el.type == 'email') || (el.type == 'url') || (el.type == 'password') || (el.type == 'tel') || (el.type == 'date')) {
+          if (['text','textarea','search','number','email','url','password','tel','date'].includes(el.type)) {
```

Comment:
You can extract this array to the outside of this function so you avoid recreating it everytime it's called 😄